src/util/Platform.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Utils for Platform.ts.
+ *
+ * @license
+ * Copyright 2023 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { networkInterfaces } from 'os';
+
+export class Platform {
```

Comment:
I don't see anything Platform specific in this file: this should work on any platform supporting node.js.

I would recommend to add listInterfaces as part of Network.ts with node.js and fake implementation so this code can be unit tested